.github/workflows/on-pullrequest-dockerfile.yml

```diff
@@ -31,8 +31,8 @@ jobs:
       matrix:
         dockerfile:
           [
-            { name: "dms_database", path: "src/backend/Dockerfile" },
-            { name: "dms", path: "src/Dockerfile" }
+            { name: "dms_database", path: "src/backend/Dockerfile", postgres_user: postgres, postgres_password: P@ssw0rd },
+            { name: "dms", path: "src/Dockerfile" , postgres_user: postgres, postgres_password: P@ssw0rd }
```

Comment:
Why is this needed, @CSR2017 ? Is there any possibility that this is creating a default password in the postgresql image? We wouldn't want that. I have not used these arguments locally when I run `docker build`.

Reply:
@stephenfuqua 
We had those values hardcoded on the docker file. So, thought of passing them as arguments to docker file. But I see your point. Will remove those.